nikola/data/themes/bootstrap3-jinja/assets/css/theme.css

```diff
@@ -175,3 +175,24 @@ article.post-micro {
 .entry-summary {
   margin-top: 1em;
 }
+
+/* hat tip bootstrap/html5 boilerplate */
+@media print {
```

Comment:
Done: `Junicode, Garamond, serif` (Junicode is an open-source font which is quite similar to Garamond and probably has similar savings in store).

Using multiple files is not really helpful.  Bundles are still the default, and even if they weren’t, that’s a wasted HTTP request.